Makefile

```diff
@@ -125,8 +132,7 @@ CONTAINERIZED= mkdir -p .go-pkg-cache $(GOMOD_CACHE) && \
 		-e KUBECONFIG=/go/src/$(PACKAGE_NAME)/kubeconfig.yaml \
 		-w /go/src/$(PACKAGE_NAME) \
 		--net=host \
-		$(EXTRA_DOCKER_ARGS) \
-		$(CALICO_BUILD)
+		$(EXTRA_DOCKER_ARGS)
```

Comment:
Is there a reason you didn't just add the CGO_ENABLED to EXTRA_DOCKER_ARGS? I think that would have avoided a lot of these changes. I know it isn't necessary for most calls but would there have been a downside?

Reply:
enabling cgo for the other builds means the compiles binaries aren't static and rely on certain libraries being present in the environment when running the compiled code.

Maybe it would work, but doesn't seem worth the extra complexity for things like UTs, etc. Also, this aligns the makefile more closely with the rest of the projectcalico makefiles, which generally have a split between the docker run command and the image being run.